src/Avalonia.Controls/TransitioningContentControl.cs

```diff
@@ -36,6 +36,14 @@ public class TransitioningContentControl : ContentControl
             nameof(IsTransitionReversed),
             defaultValue: false);
 
+  /// <summary>
```

Comment:
Indent size is wrong it should be 4 spaces, as specified in the .editorconfig (I'm adding only one comment about this, but all added code is affected).